e2e/tests/component-preview.spec.ts

```diff
@@ -6,122 +6,139 @@ const test = base.extend({
   },
 })
 
-test.describe('component preview', () => {
-  test.beforeEach(async ({ page }) => {
-    await page.goto('/components/back-link/')
-  })
-
-  test('hides the tab contents on load', async ({ page, tabRole }) => {
-    await expect(page.getByRole('tabpanel')).toHaveCount(0)
-    await expect(page.getByRole(tabRole, { name: 'HTML' })).toHaveAttribute('aria-expanded', 'false')
-    await expect(page.getByRole(tabRole, { name: 'Nunjucks' })).toHaveAttribute('aria-expanded', 'false')
-  })
-
-  test('links to the example in a new tab', async ({ page }) => {
-    await expect(page.getByRole('link', { name: 'Open this back link example in a new tab' })).toHaveAttribute(
-      'href',
-      'preview/default',
-    )
-  })
-
-  test.describe('when the HTML tab is clicked', () => {
-    test.beforeEach(async ({ page, tabRole }) => {
-      const htmlTabButton = page.getByRole(tabRole, { name: 'HTML' })
-      await htmlTabButton.click()
-      await htmlTabButton.blur()
-      await page.mouse.move(0, 0)
+const pages = [{
+  url: '/components/back-link/',
+  describe: 'component preview',
+}, {
+  url: '/components/phase-banner/',
+  describe: 'component iframe preview',
+}]
+
```

Comment:
I would have used footer here but the test was failing because `getByRole` was returning multiple elements in the footer page. 
I found that the phase-banner is responsive (subtle text font size) and this only has one example